scripts/push-dev.sh

```diff
@@ -3,6 +3,6 @@ set -e
 
 # Login with `oc login ...` before running.
 oc project officehours-dev
-docker login -u `oc whoami` -p `oc whoami -t` docker-registry.webplatformsunpublished.umich.edu
-docker build --platform linux/amd64 -t docker-registry.webplatformsunpublished.umich.edu/officehours-dev/officehours .
-docker push docker-registry.webplatformsunpublished.umich.edu/officehours-dev/officehours
+docker login -u `oc whoami` -p `oc whoami -t` registry.aws.web.umich.edu
+docker build --platform linux/amd64 -t registry.aws.web.umich.edu/officehours-dev/officehours .
+docker push registry.aws.web.umich.edu/officehours-dev/officehours
```

Comment:
Might be worthwhile to put the hostname, if not the complete URI, in a variable and reuse it on each line.  Simplifies maintenance, reduces possibilities of typos, etc.

Reply:
This should never change unless we move to a different environment. It might be worthwhile to combine and clean up these scripts but I think that's a different issue. I think maybe it could be configured in there.